### DISCUSSION TITLE
Solicitação de acesso ao Pretalx talks.python.org.br para organização da Python Nordeste 2025

### DISCUSSION THREAD

**Discussion by caiovictormc** [apyb/comunidade#172] [Pergunta]
Oi, pessoal! Estou ajudando na organização da Pyne 2025. Estamos começando a testar as ferramentas para ingressos e submissão de atividades. A @naanadr mencionou que a comunidade tem uma instância do Pretalx (talks.python.org.br). Gostaríamos de solicitar acesso para o e-mail nordeste@python.org.br.

**Comment by LanusseMorais** [comment]
@caiovictormc Criado. Você deve receber um convite no e-mail nordeste@python.org.br, esse user é admin então ele pode convidar outros membros. Para solicitações futuras referente ao pretalx você pode abrir uma issue no repo [comunidade-python-br/talks](https://github.com/comunidade-python-br/talks/issues)
